Vic2ToHoI4/Source/HOI4World/OperativeNames/OperativeNamesFactory.cpp

```diff
@@ -0,0 +1,28 @@
+#include "OperativeNamesFactory.h"
+#include "OSCompatibilityLayer.h"
+#include "OperativeNamesSetFactory.h"
+
+
+
+std::unique_ptr<HoI4::OperativeNames> HoI4::OperativeNames::Factory::getOperativeNames(const std::string& HoI4Path)
+{
+	auto operativeNames = std::make_unique<OperativeNames>();
+	OperativeNamesSet::Factory operativeNameSetFactory;
+
+	for (const auto& filename: Utils::GetAllFilesInFolder(HoI4Path + "/common/units/codenames_operatives"))
+	{
+		auto operativesNamesSet = operativeNameSetFactory.getOperativeNamesSetFromFile(
+			 HoI4Path + "/common/units/codenames_operatives/" + filename);
+		operativeNames->operativeNamesSets.insert(
+			 std::make_pair(operativesNamesSet->getWrapper(), std::move(*operativesNamesSet)));
+	}
+	for (const auto& filename: Utils::GetAllFilesInFolder("DataFiles/CodenamesOperatives"))
```

Comment:
Will this overwrite vanilla ones, or just supplant them? Just checking for intent.

Reply:
Just adds to them. It shouldn't replace them.